release_rc.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Fetch tags from upstream first
+git fetch -t git@github.com:klaytn/klaytn.git
+
+VERSION=$(go run build/rpm/main.go version)
+echo "Trying to tag v$VERSION"
```

Comment:
```suggestion
echo "Trying to tag $VERSION"
```
The tag already has "v"

Reply:
nice catch! thanks!